wrappers/python/nntile/layer/attention.py

```diff
@@ -823,16 +847,36 @@ def _forward_attn_async(self):
 
     def _forward_attn_dynamic(self, q, k, v):
```

Comment:
swap all `zeros` for temporary buffers into `empty`

Reply:
done